.coderabbit.yaml

```
@@ -0,0 +1,3 @@
+reviews:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)